docs/sites.yml

```diff
@@ -10772,6 +10772,31 @@
     - Food
   built_by: Ramón Chancay
   built_by_url: https://ramonchancay.me/
+- title: Pittica
+  url: https://pittica.com
+  main_url: https://pittica.com
+  source_url: https://github.com/pittica/site
+  description: >
+    Digital agency site.
+  categories:
+    - Design
+    - Web Development
+    - Agency
+  built_by: Lucio Benini
+  built_by_url: https://pittica.com
+  featured: false
+- title: MyHumus
+  url: https://myhumus.com
+  main_url: https://myhumus.com
+  source_url: https://github.com/my-humus/site
+  description: >
+    Digital agency site.
+  categories:
+    - Blog
+    - Food
+    - Netlify
```

Comment:
Looks like Danger bot is worried about this, Netlify must not be a tag we have on our whitelist: https://github.com/gatsbyjs/gatsby/blob/master/docs/categories.yml
```suggestion
```